openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -107,6 +107,11 @@ jQuery(function () {
         import(/* webpackChunkName: "user-website" */ './edit')
             .then(module => module.initWorksMultiInputAutocomplete());
     }
+    // conditionally load for author merge page
+    if (document.querySelector('#author-merge-page')) {
+        import(/* webpackChunkName: "user-website" */ './merge')
```

Comment:
webpack chunk name should be unique, otherwise they get merged together. Was that the intention here, or because of a copy/paste?

Reply:
It was just a copy/paste but I removed it now.